encrypt.go

```diff
@@ -1,16 +1,19 @@
 package cloudconfig
 
 type CompactTLSAssets struct {
-	APIServerCA     string
-	APIServerKey    string
-	APIServerCrt    string
-	WorkerCA        string
-	WorkerKey       string
-	WorkerCrt       string
-	CalicoClientCA  string
-	CalicoClientKey string
-	CalicoClientCrt string
-	EtcdServerCA    string
-	EtcdServerKey   string
-	EtcdServerCrt   string
+	APIServerCA       string
+	APIServerKey      string
+	APIServerCrt      string
+	WorkerCA          string
+	WorkerKey         string
+	WorkerCrt         string
+	ServiceAccountCA  string
```

Comment:
Even though we're just using the keypair I'm adding the CA and cert so they are downloaded and decrypted. That feels correct but not 100% sure. WDYT?